.buildkite/pipeline.yml

```diff
@@ -0,0 +1,107 @@
+definitions:
+  terraform_docker: &terraform_docker
+    docker#v5.11.0:
+      image: "hashicorp/terraform:1.13"
+      workdir: "/workdir"
+      entrypoint: "/bin/sh"
+  tflint: &tflint
+    docker#v5.11.0:
+      image: "ghcr.io/terraform-linters/tflint:latest"
```

Comment:
Pin tflint. We can use Renovate to keep it up to date without rolling the dice each build by pulling `latest`.